spec/unit/provider/group/solaris_spec.rb

```diff
@@ -48,7 +48,7 @@
       platforms = {
         "solaris2" => [ "-G" ],
       }
-
+      
```

Comment:
this is probably going to fail chefstyle

Reply:
it did indeed